c2corg_ui/static/js/search.js

```diff
@@ -177,14 +184,62 @@ app.SearchController.prototype.createDataset_ = function(type, title) {
     limit: 20,
     templates: {
       header: (function() {
-        return '<div class="header ' + title + '">' +
-            this.gettextCatalog_.getString(title) + '</div>';
+        return '<div class="header" dataset="' + type + '">' + this.gettextCatalog_.getString(type) + '</div>';
       }).bind(this),
+      footer: function() {
+        return '<p class="suggestion-more"><a href="/' + type + '" class="green-text">+ see more results</a></p>'
+      },
       suggestion: function(doc) {
-        return '<p>' + doc.label + '</p>';
+        var template;
+        // routes
+        if (doc.documentType === 'routes') {
```

Comment:
Do we really want to have a bunch of HTML code in the javascript code?
Can't we use a partial instead? https://github.com/c2corg/v6_ui/tree/master/c2corg_ui/static/partials